images/scripts/opensuse-tumbleweed.setup

```diff
@@ -114,6 +115,11 @@ fi
 
 zypper dup -y
 
+# cloud-init setups the wheel group incorrectly so we need to recreate it
+groupdel wheel
+zypper install -y system-group-wheel
+sudo usermod -a -G wheel admin
```

Comment:
This also feels like working around the OS, and should get a bug reference. In what way is "wheel" wrong? I.e. in a few months time how can developers decide if this can be dropped?